.github/workflows/prepare-release.yml

```diff
@@ -0,0 +1,79 @@
+name: Prepare release
+
+on:
+  workflow_dispatch:
+
+jobs:
+  prepare-release:
+    runs-on: ubuntu-latest
+
+    env:
+      GITHUB_TOKEN: ${{ secrets.TREZOR_BOT_TOKEN }}
+      FILEPATH: ./packages/suite/package.json
+      MAIN_BRANCH: develop
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          ref: ${{ env.MAIN_BRANCH }}
+
+      - name: Calculate versions
+        id: calculate-versions
+        run: |
+          currentYear=$(date +%y)
+          currentMonth=$(date +%-m)
```

Comment:
freeze is not always in the same month as the release. Maybe it makes sense to read what is the current suiteVersion? Or have it as a parameter for the action?

Reply:
Ah, right. I thought reading the value would not work in case a release was skipped. So maybe:
- read previous value rather than current month
- provide an input field to use in case the version should be set manually

Reply:
Alternatively: check the date and use the next month if it is over a certain day of the month, e.g. 25.

Reply:
it's easy to update if we ever skip the release, so I believe just reading the value from suiteVersion is enough

Reply:
fe836abbb100e04f38715d61256cef0632655778

Reply:
nice 👍